test/internal/build_settings/calculate_module_name_tests.bzl

```diff
@@ -1,6 +1,8 @@
 """Tests for module name build setting functions."""
 
 load("@bazel_skylib//lib:unittest.bzl", "asserts", "unittest")
+
+# buildifier: disable=bzl-visibility
```

Comment:
I see you opened an issue about this. I thought you could use internal in the same repo though 🤔

Reply:
In my experience, it looks like you can use it one level up. In other other repos, I have worked through this by declaring them in the parent, in a separate file from the public APIs. Then, I declare a `bzl_library` target with the visibility that I want. It is something to consider. 🤷‍♂️